tests/ztest/zexpect/src/main.cpp

```diff
@@ -1 +0,0 @@
-main.c
```

Comment:
I thought Linux symlinks were transferable to Windows but I suppose not.

Go ahead and remove the symlink as you planned but add a `main.cpp` file with `#include "main.c"` in the top. Essentially, we'll just directly include the code instead of symlinking tot he same source. That should be a filesystem independent solution.

Reply:
Yeah, i've updated